css-canvas-medical/styles.css

```diff
@@ -0,0 +1,307 @@
+.container {
+  width: 100%;
+  margin: auto;
+  max-width: 1200px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.row5 {
+  padding-top: 50px;
+}
+
+.img1-canvas {
+  margin-top: 30px;
+}
+
+.row-1 {
+  flex-direction: column;
+  width: 100%;
+  padding-top: 50px;
+  padding-left: 50px;
+}
+
+.clin {
+  padding-top: 50px;
+  padding-bottom: 70px;
+}
+
+.clin-marg {
+  padding-top: 60px;
+}
+
+h2 {
+  padding-left: 5px;
+}
+
+.font-a3 {
+  font-size: 15px;
+  padding-right: 5px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 550;
+}
+
+.font-a {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+#font-ready {
+  font-family: "Tiempos Headline", sans-serif;
+  font-style: normal;
+  font-weight: 575;
+  font-size: 25px;
+}
+
+.font-a-big {
+  font-family: "Tiempos Headline", sans-serif;
+  font-style: normal;
+  font-weight: 575;
+  font-size: 32px;
+}
+
+.font-size-large {
+  font-size: 40px;
+}
+
+.font-abs1 {
+  position: absolute;
+  top: 40px;
+  padding-left: 40px;
+  width: 55%;
+  font-family: "Tiempos Headline", sans-serif;
+  font-style: normal;
+  font-weight: 575;
+}
+
+.font-abs2 {
+  position: absolute;
+  bottom: 120px;
+  left: 40px;
+  width: 55%;
+  font-size: 19px;
+}
+
+.font-abs3 {
+  position: absolute;
+}
+
+.sch-font {
+  font-size: 15px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+}
+
+.row5a {
+  padding-top: 70px;
+  padding-bottom: 80px;
+  padding-right: 50px;
+}
```

Comment:
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.